styles.css

```diff
@@ -89,11 +122,635 @@
 
 .social-logo {
   color: #dbd8d7;
-  width: 50px;
-  height: 50px;
-  margin-top: 20px;
+  width: 15px;
+  height: 15px;
+  margin-top: 10px;
 }
 
 .logo {
-  width: 7px;
+  width: 5px;
+}
+
+.logon {
+  width: 10px;
+}
+
+.works {
+  font-family: "Crete Round", serif;
+  font-size: 30px;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #1c1a19;
+  margin-bottom: 15px;
+}
+
+.work-section {
+  color: #e5e5e5;
+  padding: 100px 25px 25px;
+}
```

Comment:
This section doesn't quite look like the template design in the desktop version.

Here's your design:

<img width="514" alt="Screen Shot 2022-05-30 at 15 17 21" src="https://user-images.githubusercontent.com/82426682/171055069-70eb5d90-964c-43f1-b363-dcaecf2a0f60.png">

<img width="514" alt="Screen Shot 2022-05-30 at 15 18 00" src="https://user-images.githubusercontent.com/82426682/171055123-04d11632-3b9d-466e-9b68-c779b1751739.png">

Here's the template design:

<img width="436" alt="Screen Shot 2022-05-30 at 15 19 56" src="https://user-images.githubusercontent.com/82426682/171055292-03460360-436b-48e5-adf0-d9d328d4cfb9.png">

Do you see how different they are? Here are some things to consider when fixing the design:

- Margins
- Paddings
- Font size and font style
- Borders